tests/pipeline/test_post_data_to_s3.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import os
+import boto3
+import zipfile
+from moto import mock_s3
+
+from neuron_morphology.pipeline.post_data_to_s3 import post_object_to_s3, zip_files
+
+path_to_current_file = os.path.realpath(__file__)
+current_directory = os.path.split(path_to_current_file)[0]
+
+file_name1 = "test_post_data_to_s3.py"
+file_name2 = "__init__.py"
+
+file_path1 = os.path.join(current_directory, file_name1)
+file_path2 = os.path.join(current_directory, file_name2)
+
+file_dict = {file_name1: file_path1, file_name2: file_path2}
+bucket_name = "neuronmorphologypipeline"
+archive_name = "test_post_data.zip"
+region = "us-west-2"
+
+@mock_s3
+def test_post_data_to_s3(tmpdir_factory):
+    
+    temp_output_dir = str(tmpdir_factory.mktemp("test"))
+    temp_output_file = os.path.join(temp_output_dir, archive_name)
+
+    s3 = boto3.resource('s3', region_name=region)
+    s3.create_bucket(Bucket=bucket_name)
+
+    archive_data = zip_files(file_dict)
```

Comment:
It would be good to add a separate test for the `zip_files` function and make this test independent of the zipping functionality.

Reply:
added the unit test for zip_files func.